app/models/container_deployment_custom_setting.rb

```diff
@@ -0,0 +1,3 @@
+class ContainerDeploymentCustomSetting < ApplicationRecord
+  belongs_to :container_deployment
+end
```

Comment:
What is the purpose of this table?  Couldn't the existing custom_settings table be leveraged?  The new table is practically identical to the existing one.